wasm-runtime/polkadot/src/support/function.rs

```diff
@@ -40,6 +42,8 @@ impl Function {
 			x if x == Function::StakingTransfer as u8 => Some(Function::StakingTransfer),
 			x if x == Function::SessionSetKey as u8 => Some(Function::SessionSetKey),
 			x if x == Function::TimestampSet as u8 => Some(Function::TimestampSet),
+			x if x == Function::GovernancePropose as u8 => Some(Function::GovernancePropose),
```

Comment:
Why not
```
let functions = [Function::StakingStake, ...];
if value as usize < functions.len() {
  Some(functions[value as usize]);
} else {
  None
}
```
doesn't it need `#[repr(u8)]` anyway for that to be safe?

Reply:
yes (done in latest PR)